test-partner/Dockerfile.debug-partner

```diff
@@ -0,0 +1,9 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+RUN \
+    yum -y install pciutils util-linux net-tools && \
```

Comment:
Is the idea to run platform test on minikube? If not, the platform already has these tools for use via chroot /host.

Reply:
Right, we want to be able to run on minikube, too. If we switched all diagnostic tests to chroot /host we could probably use a plain ubi8 image. Currently we're still using lspci/lscpu/lsblk from the container itself.

Reply:
If we start running it on non ocp clusters, we have no control over what will be available on host. It would be better to install and use the tools in the image we control?

Reply:
Agreed, for now it doesn't hurt to have those tools in the container.